uint16 header   
uint16 rotation 
uint8[96] data
